gp_emulator/gpu/computeTranspose.cu

```diff
@@ -0,0 +1,23 @@
+#include "gpu_predict.h"
+//ld_in is the leading dimension of input matrix
+//ld_out is the leading dimension of the output matrix
+extern "C"{
+real *computeTranspose(const real *matrix, const  int ld_in, const  int ld_out)
+{
+    int x, y;
+    real * temp;
```

Comment:
`result` is a better name for this variable than `temp`. Because it says what the temp variable is for.